aireview/justfile

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env just --justfile
 
```

Comment:
**documentation**: Removal of 'set dotenv-load' might break environment variable loading. Consider documenting the reason for removal or ensuring environment variables are handled through alternative means.